apps/docs/pages/docs/api-reference/fields/checkbox.mdx

```diff
@@ -0,0 +1,92 @@
+import { ConfigPreview } from "@/docs/components/Preview";
+
+# Checkbox
```

Comment:
That works, I guess it never crossed my mind as it goes against simple form standards 😅

Reply:
Have you thought about swapping out the entire form section for an npm package that can render forms from json schema? There's plenty of packages out there that can handle validation and dynamic form values based on other field values, it might make sense to swap it out for something already established as a peer dependency 

Just a thought, I ran into another problem this morning where I had a field called type, and then a custom field, and I wanted to render a set of fields based on the type value, I got this to work in edit mode but not the client render mode, but all this is supported in many other form packages, just a thought!